src/NServiceBus.Core/obsoletes.cs

```diff
@@ -1178,7 +1178,7 @@ public class MsmqDequeueStrategy
     }
 }
 
-namespace NServiceBus.Unicast.Transport
+namespace NServiceBus.Unicast.Transports
```

Comment:
Scratch my comments

Previously we had a pluralized namespace

https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/obsoletes.cs#L785

as part of this PR we changed it to `Transport`. This commit reverts it back where necessary to `Transports`.